praw/models/reddit/redditor.py

```diff
@@ -8,6 +9,15 @@
 from .base import RedditBase
 from .mixins import FullnameMixin, MessageableMixin
 
+Comment = TypeVar("Comment")
+Multireddit = TypeVar("Multireddit")
+Reddit = TypeVar("Reddit")
+_Redditor = TypeVar("_Redditor")
```

Comment:
Why does `Redditor` have an underscore prefix. Also, if it's necessary, let's define the underscore ones before the non-underscored ones.

Reply:
I’ll move it around.